iframe/priv/static/iframe/v3.html

```diff
@@ -80,6 +81,12 @@
           handleSync(callback) {
             state.syncHandler = callback;
           },
+
+          selectSecret(callback, options) {
+            state.secretHandler = callback;
+            const currentSecret = options.currentSecret || ""
+            postMessage({ type: "selectSecret", currentSecret });
```

Comment:
Let's send all options in the message, this way when we add more options we may not nee to create a new version of the iframe.